src/main/kotlin/no/nav/helsearbeidsgiver/pdl/PdlClient.kt

```diff
@@ -73,6 +77,28 @@ class PdlClient(
                 }
             }
 
+    suspend fun personBolk(identer: List<String>): List<FullPerson>? =
+        PdlQuery(personBolkQuery, Variables(identer = identer))
+            .execute(PersonBolkResultat.serializer())
+            ?.hentPersonBolk?.mapNotNull {
+                if (it.code.equals("ok", ignoreCase = true)) {
+                    val navn = it.person?.navn?.firstOrNull()
+                    val foedsel = it.person?.foedsel?.firstOrNull()
+                    if (navn == null || foedsel == null) {
+                        null
+                    } else {
+                        FullPerson(
+                            navn = PersonNavn(navn.fornavn, navn.mellomnavn, navn.etternavn),
+                            foedselsdato = foedsel.foedselsdato,
+                            ident = it.ident,
+                        )
+                    }
+                } else {
+                    logger.warn("Fikk kode ${it.code}, kunne ikke finne ${it.ident}")
```

Comment:
Er det ikke bedre å kaste exception?
Eller er tanken at hvis man ber om 2 identer men bare finner 1, så returneres bare en?

Reply:
Ja, her er ideen at vi ikke er så strenge - I dagens løsning defaulter vi allerede til "Ukjent" dersom vi ikke klarer å finne / gjøre oppslag. Tenker at det er OK å bare benytte seg av det vi får tilbake, et navn er bedre enn ingen.. :) Evt kan vi håndtere det i Simba, om vi vil være strengere på dette. 
Ved litt annet usecase for denne klienten kan man også tenke seg at man sender inn en lang liste med identer for å slå opp navn, og så er det noen som er ugyldige / ikke eksisterer - da vil jeg likevel gjerne få tilbake navn på de som går ok.

Reply:
skjønner. Da er det greit tenker jeg 👍